tests/slicetest.ark

```diff
@@ -0,0 +1,11 @@
+{
+    (import "Slice.ark")
+    # --------------------------
+    #           Slice
+    # --------------------------
+    (let slice-tests (fun () {
+        (assert (= "ABC" (sliceStr "ABCDEFG" 0 3)) "Slice test 1 failed")
+        (assert (= "1 1 1" (sliceStr "1 1 1 2" 0 5)) "Slice test 2 failed")
```

Comment:
should add tests to see if it works with `(sliceStr "a" 0 1)` for example, also `(sliceStr "abcdef" 4 1)` and `(sliceStr "abcdef" 4 12)` (must work)